runtime/minor_gc.c

```diff
@@ -512,6 +512,8 @@ void caml_empty_minor_heap_domain_clear (caml_domain_state* domain, void* unused
   clear_table ((struct generic_table *)&minor_tables->ephe_ref);
   clear_table ((struct generic_table *)&minor_tables->custom);
 
+  Caml_state->extra_heap_resources_minor = 0.0;
```

Comment:
Please can we uses the `domain` variable over `Caml_state`, we're trying to keep the uses of `Caml_state` to only those places where necessary.